go/kbnm/host_json.firefox.template

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "io.keybase.kbnm",
+  "description": "Keybase Native Messaging API",
+  "path": "@@HOST_PATH@@",
+  "type": "stdio",
+  "allowed_extensions": [
+    "keybase@keybase.io"
```

Comment:
Did we figure out whether this meant anyone could submit an extension pretending to be us?